Sources/SpeziQuestionnaire/ORKOrderedTaskView.swift

```diff
@@ -35,10 +35,12 @@ struct ORKOrderedTaskView: UIViewControllerRepresentable {
                 
                 switch reason {
                 case .completed:
-                    let fhirResponse = taskViewController.result.fhirResponse
-                    fhirResponse.subject = Reference(reference: FHIRPrimitive(FHIRString("My Patient")))
-                    
-                    await questionnaireResponse(fhirResponse)
+                    // add a for loop to check for the ORKTimedWalkResult
```

Comment:
I had some syntactical trouble implementing this for loop. Any tips on resolving this?

<img width="1036" alt="Screenshot 2023-10-04 at 1 17 23 AM" src="https://github.com/StanfordSpezi/SpeziQuestionnaire/assets/46211285/332de8b4-3931-4e2e-80b1-3530e1bac61d">

Reply:
You will probably have to add a guard statement before that first unwraps the optional value into e.g. a `results` constant and then use that in the loop 👍